cves/CVE-2017-7659.yml

```diff
@@ -47,7 +47,9 @@ description_instructions: |
   that outsiders to Chromium would not understand. Technology like "regular
   expressions" is fine, and security phrases like "invalid write" are fine to
   keep too.
-description:
+description: |
+  HTTP/2 requests could be crafted in a way that could cause mod_http2 to
```

Comment:
Maybe clarify what mod_http2 is (ie: package, file, module, etc...)

Reply:
That might be helpful, good point!